__tests__/Hospital-Detail /HospitalDetailCard.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import HospitalDetailCard, { HospitalCardProps } from '@/components/hospital-detail/HospitalDetailCard';
+
+const mockData: HospitalCardProps = {
+  image: "https://example.com/path/to/image.jpg",
+  name: "Hospital Name",
+  status: "Open",
+  logo: "https://example.com/path/to/logo.jpg",
+};
+
+describe("HospitalDetailCard", () => {
+  it("should render the hospital card with the correct data", () => {
+    const { getByAltText, getByText } = render(<HospitalDetailCard {...mockData} />);
+
+    const hospitalImage = getByAltText("Hospital Image");
+    expect(hospitalImage).toBeInTheDocument();
+
+    const hospitalLogo = getByAltText("Logo of the Hospital");
+    expect(hospitalLogo).toBeInTheDocument();
+
+    const hospitalNamePart1 = getByText("Hospital", { exact: false });
```

Comment:
using {exact:false} makes a test too generic. what are you testing with it? If you need case insensetivity you can use something like 
`  const hospitalNamePart1 = screen.getByText(/"Hospital"/i);`